kevm-pyk/src/kevm_pyk/solc_to_k.py

```diff
@@ -482,16 +512,75 @@ def _range_predicate(term, type_label: str):
         return KEVM.range_uint(256, term)
     if type_label == 'int256':
         return KEVM.range_sint(256, term)
-    if type_label == 'uint8':
-        return KEVM.range_uint(8, term)
-    if type_label == 'uint64':
-        return KEVM.range_uint(64, term)
-    if type_label == 'uint96':
-        return KEVM.range_uint(96, term)
-    if type_label == 'uint32':
-        return KEVM.range_uint(32, term)
     if type_label in {'bytes', 'string'}:
         return Bool.true
 
     _LOGGER.warning(f'Unknown range predicate for type: {type_label}')
     return None
+
+
+def _range_predicate_uint(term, type_label: str):
+    if type_label == 'uint248':
+        return (True, KEVM.range_uint(248, term))
+    if type_label == 'uint240':
+        return (True, KEVM.range_uint(240, term))
+    if type_label == 'uint232':
+        return (True, KEVM.range_uint(232, term))
+    if type_label == 'uint224':
+        return (True, KEVM.range_uint(224, term))
+    if type_label == 'uint216':
+        return (True, KEVM.range_uint(216, term))
+    if type_label == 'uint208':
+        return (True, KEVM.range_uint(208, term))
+    if type_label == 'uint200':
+        return (True, KEVM.range_uint(200, term))
```

Comment:
I think this can be replaced by logic that says

```
if type_label.startswith('uint'):
    width = int(type_label[4:])
    if not (0 < width and width <= 256 and width % 8 == 0):
        raise ValueError(f'Unsupported range predicate type: {label_type}')
    return (True, KEVM.range_uint(width, term))
```

You may be able to also make a similar simplification in `evm_base_sort` method.

Reply:
Included! If you think what I wrote looks good let's merge this :handshake: